lib/doorkeeper/oauth/refresh_token_request.rb

```diff
@@ -32,7 +33,7 @@ def load_client(credentials)
       def before_successful_response
         refresh_token.transaction do
           refresh_token.lock!
-          raise Errors::InvalidTokenReuse if refresh_token.revoked?
+          raise Errors::InvalidGrantReuse if refresh_token.revoked?
```

Comment:
What's the reason to change this class? :thinking:

Reply:
As I describe at PR's description: `3. When refresh_token is revoked`
because `refresh_token` is revoked, so we should return `invalid_grant` error as [rfc's defination](https://tools.ietf.org/html/rfc6749#section-5.2)
> invalid_grant
The provided authorization grant (e.g., authorization
code, resource owner credentials) or **refresh token is
invalid, expired, revoked,** does not match the redirection
URI used in the authorization request, or was issued to
another client.

we can see that [Errors::InvalidGrantReuse](https://github.com/doorkeeper-gem/doorkeeper/pull/1277/files#diff-f1ea26f732c79d7938f919b79036e55fL23-L25) returns `invalid_grant`, while [InvalidTokenReuse](https://github.com/doorkeeper-gem/doorkeeper/pull/1277/files#diff-f1ea26f732c79d7938f919b79036e55fL17-L21) returns `invalid_request`.

Reply:
Ahh, sorry man, too much I need to check and do after vacation :bowing_man: 
:+1: